okio/src/jvmMain/kotlin/okio/Okio.kt

```diff
@@ -108,6 +141,12 @@ private class InputStreamSource(
   override fun toString() = "source($input)"
 }
 
+fun FileChannel.store(): Store {
```

Comment:
Should be `asStore`.

"as" is for encapsulation, "to" is for transformation. We messed up with existing source and buffer functions.

Reply:
I agree. If a new convention is to be created - which it sounds like there is agreement - should the old convention be deprecated with a warning? I can create a separate pull-request for the existing functions.